app/assets/stylesheets/components/_devise.scss

```diff
@@ -0,0 +1,143 @@
+//--------------------------------------------------
+// variables
+//--------------------------------------------------
+$content-gutter: rem-calc(125);
+
+//--------------------------------------------------
+// classes
+//--------------------------------------------------
+.devise {
+  min-height: 100vh;
+
+  display: grid;
+  grid-template-columns: 7fr 5fr;
+  
+  &__panel {
+    &:last-child {
+      background-color: $green-dark;
+  
+      display: flex;
+      justify-content: center;
+  
+      position: relative;
+  
+      .devise__panel-background-image {
+        opacity: 0.16;
+        width: 100%;
+        height: 100%;
+  
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        -o-object-fit: cover;
+        object-fit: cover;
+        z-index: 1;
+      }
+    }
+  }
+
+  &__header-nav {
+    margin: rem-calc(25) rem-calc(25) 0 rem-calc(25);
+
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-link {
+      &--home {
+        color: $grey-xxdark;
+        font-family: $header-font;
+        font-size: rem-calc(18);
+        font-weight: $bold;
+        line-height: $header-font-line-height;
+        text-decoration: none;
+    
+        &::before {
+          background-image: image-url('icons/home.svg');
+          content: '';
+          margin-right: rem-calc(5);
+          width: rem-calc(15); height: rem-calc(15);
+          
+          display: inline-block;
+        }
+      }
+
+      &--other {
+        color: $green-dark;
+        line-height: $header-font-line-height;
+      }
+    }
+  }
+
+  &__content {
+    color: $grey-xxdark;
+    padding: rem-calc(185) rem-calc($content-gutter) 0 rem-calc($content-gutter);
+    max-width: rem-calc(575 + $content-gutter * 2);
+  }
+
+  &__title {
+    font-family: $header-font;
+    font-size: rem-calc(48);
+    margin-bottom: rem-calc(32);
+  }
```

Comment:
On the mocks margin here is 30, but it still looks very spced out. I reckon we can leave it is if you recon it's ok.
<img width="629" alt="image" src="https://user-images.githubusercontent.com/38047481/151451110-178fe3c2-3b1f-45bf-a307-1100e25e69e2.png">